integration-tests/gatsby-source-wordpress/__tests__/__snapshots__/index.js.snap

```diff
@@ -6301,12 +6298,6 @@ Array [
     ],
     "name": "WpNodeWithRevisionsToContentNodeConnectionEdge",
   },
-  Object {
-    "fields": Array [
-      "template",
-    ],
-    "name": "WpNodeWithTemplate",
-  },
```

Comment:
These snapshot changes are because I added `options.type.MediaItem.excludeFieldNames: [`template`]` in the test site gatsby-config

Reply:
`WpNodeWithTemplate` is now automatically excluded because one of the types that implement that interface (`WpMediaItem` in this test site) has excluded the only field that interface defines: `template`.
Any type that implements an interface must have every field in that interface. So to prevent errors, this type must also be excluded as a result of excluding `MediaItem.template`.